edge/test/integration/appdeployment/application_test.go

```diff
@@ -276,7 +276,7 @@ var _ = Describe("Application deployment in edge_core Testing", func() {
 		It("TC_TEST_APP_DEPLOYMENT_16: Test application deployment with container network configuration as port mapping", func() {
 			//Generate the random string and assign as a UID
 			UID = "deployment-app-" + edge.GetRandomString(10)
-			port := []v1.ContainerPort{{HostPort: 10255, ContainerPort: 10255, Protocol: v1.ProtocolTCP, HostIP: "127.0.0.1"}}
+			port := []v1.ContainerPort{{HostPort: 10256, ContainerPort: 10256, Protocol: v1.ProtocolTCP, HostIP: "127.0.0.1"}}
```

Comment:
why is this changed ?

Reply:
10255 is used by edge_core, we can't reuse as container HostPort